modules/mri_violations/jsx/mri_protocol_violations_columnFormatter.js

```diff
@@ -0,0 +1,31 @@
+/* exported formatColumn */
```

Comment:
Rename this and compiled file to `columnFormatterProtocolViolation.js` to correspond to the existing format